docker-compose.yml

```diff
@@ -3,15 +3,18 @@ services:
 
   devicehub:
     init: true
-    image: dkr-dsg.ac.upc.edu/ereuse/devicehub:testing_docker__f42b2d09
+    container_name: devicehub
+    build:
+      context: .
+      dockerfile: docker/devicehub.Dockerfile
     environment:
       - DB_USER=${DB_USER}
       - DB_PASSWORD=${DB_PASSWORD}
       - DB_HOST=postgres
       - DB_DATABASE=${DB_DATABASE}
       - HOST=${HOST}
-      - EMAIL_DEMO=${EMAIL_DEMO}
-      - PASSWORD_DEMO=${PASSWORD_DEMO}
+      - EMAIL_DEMO=usuario@ekoa.com
+      - PASSWORD_DEMO=ekoa
```

Comment:
Nice changes, but I would keep the `${EMAIL_DEMO}` and `${PASSWORD_DEMO}` here. :smile: